specification/redis/resource-manager/Microsoft.Cache/stable/2021-06-01/redis.json

```
@@ -1933,6 +1933,10 @@
               "description": "The max clients config",
               "readOnly": true
             },
+            "notify-keyspace-events": {
+              "type": "string",
+              "description": "The KeySpace Events which should be monitored."
```

>The KeySpace Events [](http://example.com/codeflow?start=30&length=19)

```suggestion
the keyspace events
```
doc style capitalization